lesson_09/homeworks.py

```diff
@@ -0,0 +1,69 @@
+
+""" Задача - надрукувати табличку множення на задане число, але
+лише до максимального значення для добутку - 25.
+Код майже готовий, треба знайти помилки та випраавити\доповнити.
+"""
+
+def multiplication_table(number=None):
+    if number is None:
+        return ("The input must be a positive integer.")
+    if not isinstance(number, int):
+        return ("The input must be a positive integer.")
+    if number < 1:
+        return ("The input must be a positive integer.")
+    table = []
+    multiplier = 1
+    while True:
+        result = number * multiplier
+        if  result > 25:
+            break
+        table.append(str(number) + "x" + str(multiplier) + "=" + str(result))
+        multiplier += 1
+    return table
+
+"""  Написати функцію, яка розрахує середнє арифметичне списку чисел."""
+
+def list_avg(lister=None):
+    if lister is None or not isinstance(lister, list) or not lister:
+        return "The input must be list."
+    sum_list = 0
+    count = 0
+    for x in lister:
+        if not isinstance(x, (float, int)):
+            return "The number in list must be float."
+        sum_list += x
+        count += 1
```

Comment:
```suggestion
        count += 1
    if count == 0:
        raise ValueError("Cannot compute average of an empty list.»)
```
Для випадку передачi пустого листа